### DISCUSSION TITLE
Data preprocessing module

### DISCUSSION THREAD

**Discussion by vgro** [ImperialCollegeLondon/virtual_rainforest#128] [Ideas]
We started talking about the need for preprocessing of input data, mostly climate data, to scale to our model grid. I've started working on this but we think this is something that should be done separately from the main VR and could probably be outsourced to contractors. For now, we suggest that we park this as it is not essential for the dummy version but it would be something to think about for the next stage. I could open a branch to put some of the ideas and simple docs on that or we could just leave it for now and pick up the discussion in the next phase?

**Comment by davidorme** [comment]
I think this makes a lot of sense. If we create  the new branch `feature/preprocessors` that initially simply creates the `virtual_rainforest.preprocessors` module with an `__init__.py`, then we can use `feature/preprocessors/sub_branch` to start to post existing code snippets for functions with a view to merging them in.

We should probably have some kind of a common strategy for branch names!

**Comment by davidorme** [comment]
@vgro Is this discussion now superseded by the code being collected in [ImperialCollegeLondon/ve_data_science ](https://github.com/ImperialCollegeLondon/ve_data_science/)

**Reply by vgro** [reply]
yes, I think we can close this
